internal/uvm/types.go

```diff
@@ -152,6 +152,13 @@ func (uvm *UtilityVM) ScratchEncryptionEnabled() bool {
 	return uvm.encryptScratch
 }
 
+func (uvm *UtilityVM) GetWCOWCreateOpts() *OptionsWCOW {
```

Comment:
My PR added a similar function here: https://github.com/microsoft/hcsshim/blob/main/internal/uvm/security_policy.go#L125, I think we should be able to use that here too.